### ISSUE TITLE
[HW2] API 사용 관련 질문드립니다.

### ISSUE THREAD

**Issue by jihoonrf** [swsnu/swppfall2019#68]
제공된 api의 모든 기능을 정해진 곳에 사용해야 하는지, 아니면 그중 일부만을 자유롭게 사용해도 되는지 궁금합니다.
가령 api/user/1 에 대한 GET 리퀘스트는, 정해진 한명의 유저에 대해 로그인 여부를 redux에서 잘 기록하고 있으면 이번 과제에 한해서는 굳이 쓰지 않아도 되는 기능일 것 같습니다.
이런 경우처럼 api 사용은 완전히 자유롭게 해도 괜찮을까요?

**Comment by ktaebum** [comment]
Yes, you can use it freely. Just keep in mind that your simple backend server should be always synchronized with app's current state.
